examples/material_gallery/lib/gallery/example_code.dart

```diff
@@ -0,0 +1,230 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Note: This code is not runnable, it contains code snippets displayed in the
+// gallery.
+
+import 'package:flutter/material.dart';
+
+class ButtonsDemo{
+  void setState(VoidCallback callback) {}
```

Comment:
note that `{}` is correct per our style for empty maps, while `{ }` is for empty code blocks.